addons/account/views/account_move_views.xml

```diff
@@ -198,6 +198,9 @@
             <field name="mode">primary</field>
             <field name="inherit_id" ref="account.view_move_line_tree_grouped"/>
             <field name="arch" type="xml">
+                <xpath expr="//tree" position="attributes">
+                    <attribute name="js_class">account_print_as_xlsx</attribute>
```

Comment:
There was already a js_class attribute set in enterprise here https://github.com/odoo/enterprise/blob/41c5e15600ae3be877c3dda2556467559ad3d5d0/account_accountant/views/account_move_views.xml#L9

I didnt test it myself, but from the screenshots you have on the task, that js_class isn't loaded anymore

Reply:
@william-andre 
I have tested it and its different view, so it will not affect each other. (In the spec, three views are inherited from view "view_move_line_tree_grouped", and you have applied js_class directly on "view_move_line_tree_grouped" so it will not affect).

Both are working fine.

Reply:
What do you mean? The class `account_move_line_list` should be loaded on every single one of the views extended from `view_move_line_tree_grouped`. This is not the case from what I see in this screenshot https://drive.google.com/file/d/1fOPuL-CvWFAAZw8M-xlaSQpmkcJGTNNW/view?usp=sharing

In order to make it work, the least bad option I can come up with right now is create a class in enterprise that inherits both `account_move_line_list` and `grouped_tree_print_as_xlsx`, and add that `js_class` to the 3 views, like you did in community.